guardian/shortcuts.py

```diff
@@ -348,7 +348,7 @@ def get_groups_with_perms(obj, attach_perms=False):
 
 
 def get_objects_for_user(user, perms, klass=None, use_groups=True, any_perm=False,
-                         with_superuser=True, accept_global_perms=True):
+                         with_superuser=True, accept_global_perms=False):
```

Comment:
I suggest add some dummy object and add warning if not defined explicte. This library is important in terms of security and we can not surprise its users with changes in access policies. I think it is better to introduce a warning now and force an adjustment than to be surprised by the need to adjust.
```
 def get_objects_for_user(user, perms, klass=None, use_groups=True, any_perm=False,                         with_superuser=True, accept_global_perms=DEFAULT_VALUE):
     if accept_global_perms is DEFAULT_VALUE:
          warnings.warn("The behavior of 'accept_global_perms' is reversed in django-guardian v 2.x. Define the desired behavior directly to maintain compatibility and security in the future.") 
          accept_global_perms = True  # TODO: changed in django-guardian v 2.x
```

Reply:
if users used the default value before, and make their code work. then after grade, the code may break. In this case, we should raise warnings.
but if users used a specific value before, after grade, the code could also work. we should not give warnings.
we can make these functions receive a dict, and get values from it.

code maybe like this:
```
def get_objects_for_user(user, perms, **kwargs):
    # get values from kwargs
    ....
```